os/arch/arm/src/imxrt/imxrt_mtd_flash.c

```diff
@@ -279,28 +277,22 @@ static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, unsigned long arg)
 	/* Deselect the FLASH */
 
 	SPI_SELECT(priv->dev, SPIDEV_FLASH, false);
+#endif
 	
-	size_t sectors_erased = 0;
-	size_t startsector = 8;
-//	fdbg("startsector: %d nsectors: %d\n", (long)startsector, (int)priv->nsectors);
-	fdbg("startsector: %d nsectors: %d\n", (long)startsector, 2);
+	off_t sectors_erased = 0;
+	fvdbg("priv: %p, startblock: %d, startaddr: %x, nsectors: %d\n", priv, startblock, startblock >> priv->pageshift, nsectors);
 
 	/* Lock access to the SPI bus until we complete the erase */
 
-//	while (sectors_erased < priv->nsectors) {
-	while (sectors_erased < 2) {
+	while (sectors_erased < priv->nsectors) {
```

Comment:
Before start erase, spi should be locked.

Reply:
Strictly speaking, imxrt_bulkerase is not used just to run FS TCs successfully.
So I reverted this function, this will be implemented in another PR.